test/e2e/setup.ts

```diff
@@ -10,15 +9,16 @@ export default async function setup() {
 	const page = await browser.newPage();
 
 	// Remove old state before starting
-	clearState();
+	// clearState();
```

Comment:
Can remove this from here.

Reply:
actually, forgot to uncomment this :grimacing: